caas/containers.go

```diff
@@ -55,6 +56,109 @@ type ContainerSpec struct {
 	ProviderContainer `yaml:"-"`
 }
 
+// RbacType describes Role/Binding type.
+type RbacType string
```

Comment:
RBAC is an acronym, so should be RBACType.

Reply:
right, renamed